### ISSUE TITLE
Why in the official documentation of rocm-5.6.0, the llvm-target of RX7600 is gfx1100

### ISSUE THREAD

**Issue by JiaJiDuan** [RadeonOpenCompute/ROCm#2500]
I learned through rocminfo that my RX7600 corresponds to gfx1102. But in the official documents, it is gfx1100 that is tested. Why?
[https://rocm.docs.amd.com/en/docs-5.6.0/release/windows_support.html](url)  
Is RX7600 officially supported on linux?  
What should my llvm-target set for RX7600 on a Ubuntu system?

**Comment by YellowRoseCx** [comment]
Good catch, I've added it to the others in my pull request https://github.com/RadeonOpenCompute/ROCm/pull/2497

**Comment by evshiron** [comment]
@JiaJiDuan 

> What should my llvm-target set for RX7600 on a Ubuntu system?

I am afraid treating it as `gfx1100` via `export HSA_OVERRIDE_GFX_VERSION=11.0.0` should just work.
